tests/Tests/500__jupyterhub/long-running-test-generic-ds.robot

```diff
@@ -26,6 +26,19 @@ Launch JupyterLab
   &{S3-credentials} =  Create Dictionary  AWS_ACCESS_KEY_ID=${S3.AWS_ACCESS_KEY_ID}  AWS_SECRET_ACCESS_KEY=${S3.AWS_SECRET_ACCESS_KEY}
   Spawn Notebook With Arguments  image=s2i-generic-data-science-notebook  envs=&{S3-credentials}
 
+Verify Installed Libraries in SDS
+  [Tags]  Sanity
+  Python Version Check
+  Run Keyword And Continue On Failure  Run Cell And Check Output  !pip show boto3 | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s", b[1], b[2]}'  1.17
```

Comment:
For clarity you could create a `Python Package Should Be Installed` keyword and make the call like this:
`Run Keyword And Continue On Failure     Python Package Should Be Installed    boto3   1.17`

Why do you `Run Keyword And Continue On Failure `. Don't you want to fail if requirements are not met?

Reply:
For point #1 see the other comment, point #2 no I don't want the TC to fail because of that, at least in this setup. There is other testing being done after the package version checks and it's not worth it to skip it entirely if this check fails. The failure is still shown in the run logs so we are still notified of any problems with the packages